mobile/package.json

```diff
@@ -32,10 +33,12 @@
     "expo-dev-client": "~3.3.11",
     "expo-font": "~11.10.3",
     "expo-linking": "~6.2.2",
+    "expo-localization": "~14.8.3",
     "expo-router": "~3.4.8",
     "expo-status-bar": "~1.11.1",
     "expo-updates": "~0.24.11",
     "i18next": "^23.10.1",
+    "mkdirp": "^3.0.1",
```

Comment:
This should not be here, please remove this dependency.

Reply:
done